Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -0,0 +1,191 @@
+---
+title: CephCluster Network Providers
+---
+
+!!! Note
+    This is an advanced networking topic.
+    [General CephCluster networking is linked here.](./ceph-cluster-crd.md#network-configuration-settings)
+
+Rook deploys CephClusters using Kubernetes' software-defined networks by default. This is simple for
+users, provides necessary connectivity, and has good node-level security. However, this comes at the
+expense of additional latency, and the storage network must contend with Kubernetes applications for
+network bandwidth. It also means that Kubernetes applications coexist on the same network as Ceph
+daemons and can reach the Ceph cluster easily via network scanning.
+
+Rook allows selecting alternative network providers to address some of these downsides, sometimes at
+the expense of others. Selecting alternative network providers is an advanced topic.
+
+## Ceph Networking Fundamentals
```

Comment:
Renamed from "Ceph public and cluster networks" since it seems clear that too many people skip the section. Hopefully "fundamentals" suggests that it is critical for understanding the rest of the doc.